samcli/lib/init/templates/cookiecutter-aws-sam-hello-java-gradle/tests/test_cookiecutter.py

```diff
@@ -0,0 +1,40 @@
+"""
+    Tests cookiecutter baking process and rendered content
```

Comment:
This doesn't shown as moved, is this a new file?

Reply:
Looks like the others shown as removed and this added.

Reply:
Yeah I don't know why. Probably because black did some big refactors?